syntax/index.html

```diff
@@ -587,15 +578,16 @@
                 <p>Parse the tabular data file, using <code>DD</code> as a guide, to create a basic tabular data model (<code>T</code>) and extract <a>embedded metadata</a> (<code>EM</code>), for example from the <a>header line</a>.</p>
                 <p class="note">This specification provides a non-normative definition for parsing CSV-based files, including the extraction of <a>embedded metadata</a>, in <a href="#parsing" class="sectionRef"></a>. This specification does not define any syntax for embedded metadata beyond this; whatever syntax is used, it's assumed that metadata can be mapped to the vocabulary defined in [[!tabular-metadata]].</p>
```

Comment:
Ah, yep, I found it below, sorry for the noise. 

However... then there should be some statement to signal what a processor should do if `EM` and `TM` are *not* compatible. I presume that is an error condition, meaning the processing on that particular table (or the whole table group?) should stop, or something like that. I believe this belongs to this section.